doc/src/pair_coul.rst

```diff
@@ -20,6 +20,7 @@
 .. index:: pair_style coul/msm
 .. index:: pair_style coul/msm/omp
 .. index:: pair_style coul/streitz
+.. index:: pair_style coul/ctip
```

Comment:
this list should be in alphabetic order I believe?

Reply:
just the section from debye to wolf
the cout/cut and tip4p variants are separate

Reply:
from this, i take it we'll keep coul/cut and coul/tip4p ordered as-is, and move coul/ctip to start of the post-cut styles (instead of very strictly alphabetic)? i.e.:

```
pair_style coul/cut
pair_style coul/cut/gpu
...
pair_style coul/cut/global/omp
pair_style coul/ctip
pair_style coul/debye
...
```

Reply:
yes, that would be fine